tensorflow/core/api_def/base_api/api_def_ApplyAdam.pbtxt

```diff
@@ -82,9 +82,9 @@ END
   }
   summary: "Update \'*var\' according to the Adam algorithm."
   description: <<END
-$$lr_t := \text{learning\_rate} * \sqrt{1 - beta_2^t} / (1 - beta_1^t)$$
-$$m_t := beta_1 * m_{t-1} + (1 - beta_1) * g$$
-$$v_t := beta_2 * v_{t-1} + (1 - beta_2) * g * g$$
-$$variable := variable - lr_t * m_t / (\sqrt{v_t} + \epsilon)$$
+$$\text{lr}_t := \mathrm{lr} * \frac{\sqrt{1 - \beta_2^t}}{1 - \beta_1^t}$$
```

Comment:
I have a preference for either \cdot or just not having an operator for the scalar multiplication (it isn't due to you though, and there may be an argument for consistency with the others, but looks wise asterisk is so jarring in a math formula)

Reply:
I have exactly the same preference. I think that \cdot makes it cleaner after \frac is removed.